docs/guides/permissions/aws/main.tf

```
@@ -0,0 +1,87 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.3.0"
+    }
```

what happened to the minification?

```suggestion
    aws = { source = "hashicorp/aws", version = "~> 4.3.0" }
```

---

🤦🏼‍♂️ I never applied it. It was unindented, I swear!